### DISCUSSION TITLE
Disabling font bold

### DISCUSSION THREAD

**Discussion by peguimasid** [Railly/one-hunter-vscode#61] [Show and tell]
For those who prefer not to use bold fonts, here's a quick way to disable bold in the `One Hunter Theme Vercel`. Simply add the following code to your VSCode settings, replacing the theme name as needed:

```json
"editor.tokenColorCustomizations": {
    "[One Hunter Theme Vercel]": {
      "comments": { "fontStyle": "" },
      "functions": { "fontStyle": "" },
      "keywords": { "fontStyle": "" },
      "numbers": { "fontStyle": "" },
      "strings": { "fontStyle": "" },
      "types": { "fontStyle": "" },
      "variables": { "fontStyle": "" },
      "textMateRules": [
        { "scope": "keyword.operator", "settings": { "fontStyle": "" } }
      ]
    }
  }
```

Feel free to adapt it for your preferred theme!

**Comment by Railly** [comment]
Thank you @peguimasid. I've created an issue referencing this discussion. I'll definitely add an option to toggle between bold and non-bold variants. 🤠

**Comment by CJ-cooper6** [comment]
Hi，I added settings in VSCode but some parts are still bold.
<img width="536" alt="image" src="https://github.com/Railly/one-hunter-vscode/assets/71270517/855271e2-d631-416e-901e-b8182d9c53fa">

**Comment by trungklam** [comment]
Adding some scopes to fully disable bold font https://github.com/Railly/one-hunter-vscode/issues/62#issuecomment-2140236527
